src/main/drivers/system.c

```diff
@@ -284,3 +284,21 @@ void unusedPinsInit(void)
 {
     IOTraversePins(unusedPinInit);
 }
+
+char *getMCUTypeName(void)
+{
+#if defined(STM32H7)
```

Comment:
I assume that this should be `STM32H743xx`, or else the returned strings will be incorrect for H750?